test/common.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const Code = require('@hapi/code');
+const Joi = require('@hapi/joi');
 const Lab = require('@hapi/lab');
 
 const Common = require('../lib/common');
```

Comment:
We shouldn't need to test against the public version of joi, ideally.  I am hoping that this will work just as well:

```suggestion
const Lab = require('@hapi/lab');

const Joi = require('..');
const Common = require('../lib/common');
```

Reply:
Ah, of course! Thanks, that totally makes sense 👍